java-checks/src/main/java/org/sonar/java/checks/tests/AbstractJUnit5NotCompliantModifierChecker.java

```diff
@@ -36,21 +36,27 @@
 
 public abstract class AbstractJUnit5NotCompliantModifierChecker extends IssuableSubscriptionVisitor {
 
+  public enum ModifierScope {
+    CLASS,
+    CLASS_METHOD,
+    INSTANCE_METHOD
+  }
```

Comment:
It would be very helpful to document how these enums represent and how they can be used